.github/actions/setup-node/action.yml

```diff
@@ -40,8 +40,11 @@ runs:
       with:
         timeout_minutes: 10
         max_attempts: 3
-        command: yarn install --frozen-lockfile --ignore-scripts
+        command: |
+          yarn install --frozen-lockfile --ignore-scripts
+          yarn prepare:install-re2
+          yarn prepare:re2
```

Comment:
i think we can remove this check and move it to a jest test instead to make things simpler.
Or mode the install call inside the `tools/check-re2.mjs`